index.html

```diff
@@ -41,7 +41,7 @@
     </nav>
   </header>
   <section class="page-section headline-section">
-    <h1 class="section-title title-l1">Hey There. I'm Gabriela</h1>
+    <h1 class="section-title title-l1">Hey There. <br> I'm Gabriela</h1>
     <h3 class="section-subtitle subtitle-l1">I'm a Software Developer</h3>
```

Comment:
- [ ] [OPTIONAL]: To keep the HTML readable and accessible for users of screen reading software, you could consider making this an `<h2>` instead, you should try to not skip headings. You can read more about headings [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements).

Reply:
Thank you for your review, Vanessa.
I have a question.
I am using the h2 header as a title of other sections because I read that only one h1 header must be added to a web page.
I wanted to ask you if I can put subtitles with h2, and the h3 heading as titles for other sections.
Thanks in advance.

Reply:
Hi @gabyse1, yes! You can have h2 and h3 as titles for other sections! You should only add one h1 for accessibility and SEO purposes.